<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="workers_table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>748</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="workers_tab_basicinfo">
    <attribute name="title">
     <string>基本信息</string>
    </attribute>
    <widget class="QPushButton" name="workers_table_signoutpushButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>370</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
    <widget class="QPushButton" name="workers_table_okpushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>370</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="workers_table_usergroupBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>531</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>用户信息</string>
     </property>
     <widget class="QLabel" name="workers_table_idlabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>工号</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_idlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_idlineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_namelabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_namelineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_namelineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="workers_table_basicgroupBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>基本信息</string>
     </property>
     <widget class="QLabel" name="workers_table_sexlabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>性别</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_sexlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_agelineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_personnellabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>所属部门</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_personnellineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_phonelabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>联系方式*</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_phonelineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_positionlabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>职称</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_positionlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_addresslabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>籍贯*</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_addresslineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_idcardlabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>身份证号</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_idcardlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_birthlabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>出生年月*</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_birthlineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_addresslineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_positionlineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_personnellineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="workers_table_agelabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄*</string>
      </property>
      <property name="buddy">
       <cstring>workers_table_agelineEdit</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_phonelineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_birthlineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_idcardlineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="workers_table_sexlineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="workers_tab_payinfo">
    <attribute name="title">
     <string>工资信息</string>
    </attribute>
    <widget class="QDateEdit" name="workers_table_dateEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-M</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_datelabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_totalpaylabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>总工资 </string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_taxlabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>税收</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_incomelabel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>270</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>净工资</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_totalpaylineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_taxlineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_incomelineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="workers_table_signoutpushButton_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>340</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="workers_tab_attendinfo">
    <attribute name="title">
     <string>考勤信息</string>
    </attribute>
    <widget class="QDateEdit" name="workers_table_dateEdit_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2013</year>
       <month>5</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2013</year>
       <month>5</month>
       <day>1</day>
      </date>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy-M</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_datelabel_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_latelabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>迟到次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_absentlabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>缺席次数</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_leavelabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>请假次数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_latelineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_leavelineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_absentlineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="workers_table_signoutpushButton_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>350</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注销</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="workers_tab_chpasswd">
    <attribute name="title">
     <string>修改密码</string>
    </attribute>
    <widget class="QLineEdit" name="workers_table_oldpasswdlineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_newpasswdlineEdit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="workers_table_newpasswdlineEdit_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_oldpasswdlabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>原密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_newpasswdlabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>输入新密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="workers_table_newpasswdlabel_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认新密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="workers_table_cancelpushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="workers_table_okpushButton_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="workers_hellolabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>您好！</string>
   </property>
  </widget>
  <widget class="QLabel" name="workers_datetime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>系统时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>workers_table_idlineEdit</tabstop>
  <tabstop>workers_table_namelineEdit</tabstop>
  <tabstop>workers_table_phonelineEdit</tabstop>
  <tabstop>workers_table_sexlineEdit</tabstop>
  <tabstop>workers_table_personnellineEdit</tabstop>
  <tabstop>workers_table_idcardlineEdit</tabstop>
  <tabstop>workers_table_birthlineEdit</tabstop>
  <tabstop>workers_table_agelineEdit</tabstop>
  <tabstop>workers_table_addresslineEdit</tabstop>
  <tabstop>workers_table_positionlineEdit</tabstop>
  <tabstop>workers_table_signoutpushButton</tabstop>
  <tabstop>workers_table_okpushButton</tabstop>
  <tabstop>workers_table</tabstop>
  <tabstop>workers_table_dateEdit</tabstop>
  <tabstop>workers_table_totalpaylineEdit</tabstop>
  <tabstop>workers_table_taxlineEdit</tabstop>
  <tabstop>workers_table_incomelineEdit</tabstop>
  <tabstop>workers_table_signoutpushButton_2</tabstop>
  <tabstop>workers_table_dateEdit_2</tabstop>
  <tabstop>workers_table_latelineEdit</tabstop>
  <tabstop>workers_table_leavelineEdit</tabstop>
  <tabstop>workers_table_absentlineEdit</tabstop>
  <tabstop>workers_table_signoutpushButton_3</tabstop>
  <tabstop>workers_table_oldpasswdlineEdit</tabstop>
  <tabstop>workers_table_newpasswdlineEdit</tabstop>
  <tabstop>workers_table_newpasswdlineEdit_2</tabstop>
  <tabstop>workers_table_cancelpushButton</tabstop>
  <tabstop>workers_table_okpushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
